samples/trumpify/trump.py

```diff
@@ -0,0 +1,83 @@
+import os
+import sys
+import time
+from threading import Thread
+
+from twitter import *
+
+sys.path.append(os.path.abspath('../../'))
+
+from daemo.client import Client
+
+CREDENTIALS_FILE = 'credentials.json'
+
+PROJECT_ID = -1
+
+TW_CONSUMER_KEY = ''
+TW_CONSUMER_SECRET = ''
+TW_ACCESS_TOKEN = ''
+TW_ACCESS_TOKEN_SECRET = ''
```

Comment:
that CREDENTIALS file is what users download from Daemo to interact with platform.